src/main/java/com/synopsys/integration/alert/channel/msteams/MsTeamsMessageParser.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.channel.msteams;
+
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.channel.message.ChannelMessageParser;
+import com.synopsys.integration.alert.common.message.model.MessageContentGroup;
+import com.synopsys.integration.alert.common.message.model.ProviderMessageContent;
+
+@Component
+public class MsTeamsMessageParser extends ChannelMessageParser {
+
+    @Override
+    protected String encodeString(String txt) {
+        return txt;
+    }
+
+    @Override
+    protected String emphasize(String txt) {
+        return String.format("*%s*", txt);
+    }
+
+    @Override
+    protected String createLink(String txt, String url) {
+        return String.format("[%s](%s)", txt, url);
+    }
+
+    @Override
+    protected String getLineSeparator() {
+        return "\r\n";
+    }
+
+    @Override
+    protected String getSectionSeparator() {
+        return "";
+    }
+
+    @Override
+    public String createHeader(MessageContentGroup messageContentGroup) {
```

Comment:
I don't know if this is correct. Every piece of message that we send will require this wrapping (e.g. if we split it into multiple requests). This should exist outside of the message parser.

Reply:
From what I saw, the only place has multiple is the components themselves. The header won't ever be duplicate.

Reply:
I think I see what you mean. But the way we do it is to split it out into cards. I will check tomorrow on our server to see if the cards can handle large data loads (Seamonkey). If it can, then I think it's fine as is